quicklogic/qlf_k4n8/tests/features/install_test/CMakeLists.txt

```diff
@@ -1,12 +1,42 @@
 set(INSTALLATION_DIR_BIN "${CMAKE_INSTALL_PREFIX}/bin")
 
-function(add_binary_test target_name test_name family)
-	add_test(NAME ${target_name}
-		COMMAND ${CMAKE_COMMAND} -E env
-		PATH=${INSTALLATION_DIR_BIN}:$ENV{PATH}
-		${CMAKE_COMMAND} -E env
-		make TEST=${test_name} FAMILY=${family}
-		WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
+function(add_binary_test target_name test_name family extra_args)
+
+  set(SOURCES "${test_name}.v")
+  set(PINMAP "pinmap_${family}_umc22.csv")
+
+  set(PCF "${test_name}.pcf")
+  set(SDC "${test_name}.sdc")
+
+  set(TOOLCHAIN_COMMAND "\
+    ql_symbiflow \
+    -synth \
+    -src ${CMAKE_CURRENT_SOURCE_DIR} \
+    -d ${family} \
+    -t top \
+    -v ${SOURCES} \
+    -P ${PINMAP} "
+  )
```

Comment:
Does this run also the post synthesis test?

Reply:
Right, apparently it was doing synthesis only. I've added dumping post-pnr verilog.